force_wfmanager/ui/setup/mco/kpi_specification_view.py

```diff
@@ -0,0 +1,310 @@
+from traits.api import (
```

Comment:
This file has been refactored from `kpi_specification_model_view`, though Git recognises it as a new file

Reply:
Ah, this file contains both `KPISpecificationView` and `KPISpecificationModelView` classes, which is confusing I agree. In #313 this has been inadvertently resolved (along with the MCO parameters).

Reply:
Haha good to hear! It probably doesn't help that I take so long to review everything!